src/makeRobot.js

```diff
@@ -38,7 +38,82 @@
  * @return {object}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    'name': name,
+    'wheels': wheels,
+    'version': version,
+    'coords': {
+      'x': 0,
+      'y': 0,
+    },
+  };
+
+  robot.info
+  = `name: ${robot.name}, chip version: ${
+      robot.version}, wheels: ${robot.wheels}`;
+
+  robot.location = `${robot.name}: x=${robot.coords.x}, y=${robot.coords.y}`;
```

Comment:
location и info должны быть геттерами. Если так сделать не надо будет этот код robot.location
      = `${robot.name}: x=${robot.coords.x}, y=${robot.coords.y}`; в методах goBack, goForward, goLeft, goRight. 
```suggestion
     const robot = {
    name: name,
    wheels: wheels,
    version: version,
    coords: {
      x: 0,
      y: 0,
    },
    get location() {
      return `${this.name}: x=${this.coords.x}, y=${this.coords.y}`;
    },
  };
```